lib/middleware/lambdas.js

```diff
@@ -20,4 +20,5 @@ module.exports = function lambdas(req, res) {
       };
     }
   });
+  next();
 };
```

Comment:
Use typical express middleware signature so it's open for options and for consistency's sake

```
function (options) {
   return function (req, res, next) {
       Object.assign({
       ......
       })
       next();
})
```